.ci/cloudbuild-pull-request.yaml

```
@@ -92,14 +92,33 @@ steps:
 #    - "-c"
 #    - |
 #     yarn e2e:ci --base=origin/${_BASE_BRANCH} --head=HEAD --parallel=3
-  - id: "Build the apps"
+  - id: "Build the apps" 
     name: "node:16"
     entrypoint: "bash"
+    waitFor:
+      - "Unit Tests"
+      - "Linting the code"
     args:
       - "-c"
       - |
         yarn build:ci --base=origin/$_BASE_BRANCH --head HEAD --parallel=3
-
+  
+  - id: "Find Affected Projects"
```

This is a more contextual name
```suggestion
  - id: "Build storybook"
```